tests/regression/09-regions/23-evilcollapse_rc.c

```diff
@@ -40,15 +42,15 @@ static inline void list_add(struct list_head *new, struct list_head *head) {
 }
 
 inline static struct list_head *lookup (int d) {
-  int hvalue;
+  int hvalue = __VERIFIER_nondet_int();
   struct list_head *p;
   p = c.slot[hvalue].next;
```

Comment:
I guess if you are strict about it, such things are possibly undefined behavior, as `hvalue` might go beyond the array.

We don't care about it, but I don't know if the SV-Comp folks are also so relaxed about UB.

Reply:
I noticed it in a bunch of places too. I suppose I'll have to add some assertion/assumption after it, keeping only a valid range.

Reply:
I just committed a fix to this as well using `assume_abort_if_not` defined through `abort` which is how the rest of sv-benchmarks handle such thing.